docs/source/conf.py

```diff
@@ -52,6 +52,16 @@
 add_module_names = False
 
 
+def skip_log(app, what, name, obj, skip, options):
+    if what == "attribute" and name == "log":
```

Comment:
Whoops, looks like the `log` object is considered to be a "data" type (who knew 🤷🏼‍♂️, my bad) and the "name" is the fully qualified name:

```suggestion
    if what == "data" and name.split(".")[-1] == "log":
```

Reply:
Oh wow, thanks for figuring that out!